test/core/setToken.spec.ts

```diff
@@ -390,145 +390,4 @@ contract('SetToken', accounts => {
       });
     });
   });
-
-  describe('#transfer', async () => {
```

Comment:
why are you removing these?

Reply:
It's unnecessary...it basically calls the inherited xfer fxn except has a check to make sure that the recipient address isn't the smart contract itself. Just no need to have it in my mind.